src/common/components/cards/DeckSummary.js

```diff
@@ -53,16 +54,24 @@ export default class DeckSummary extends Component {
     };
   }
 
+  get isDefaultDeck() {
+    return this.props.deck.id.startsWith('[default-');
```

Comment:
Is there a clean way we could bake the notion of default deck onto the deck object instead of relying on a naming convention? I'm thinking `this.props.deck.isDefaultDeck`.

Reply:
Right now `deck` is just a plain ol' dumb JavaScript object (like everything else in the state), but it may be a good idea at some point to transition to smarter objects for elements of state.

Reply:
Gotcha, yeah I think a Deck object would be a decent future PR. Is there any way a player could accidentally or intentionally get their deck id to match this logic, or are those IDs generated internally?

Reply:
Nah, the ids are randomly generated strings. So at the very least it would be pretty unlikely.